pkcs5/src/pbes2.rs

```diff
@@ -321,6 +369,18 @@ pub enum EncryptionScheme {
         iv: [u8; AES_BLOCK_SIZE],
     },
 
+    /// AES-128 in CBC mode
+    Aes128Gcm {
+        /// GCM nonce
+        iv: [u8; GCM_NONCE_SIZE],
+    },
+
+    /// AES-256 in GCM mode
+    Aes256Gcm {
+        /// GCM nonce
+        iv: [u8; GCM_NONCE_SIZE],
```

Comment:
Any reason to call it `iv` instead of just `nonce`?

Reply:
Nope! I'll change this

Reply:
Done